crazyflie_dock_node:
  ros__parameters:
    rate: 2.0
    robot_1: ['crazyflie_robot_1', 'radio://0/80/2M/E7E7E7E7E7']
    robot_2: ['crazyflie_robot_2', 'radio://0/80/2M/E7E7E7E7E8']